content/en/platform/corda/5.2/deploying-operating/keys/_index.md

```diff
@@ -0,0 +1,196 @@
+
+   {{< tabs >}}
+   {{% tab name="Bash"%}}
+   ```shell
+   curl -k -u $REST_API_USER:$REST_API_PASSWORD -X PUT -d '{"section":"corda.crypto", "version":"1", "config": \
+    "{\"caching\ :\"expireAfterAccessMins\":{\"default\":60},\"maximumSize\" :{\"default\":10000}},\"hsm\": \
+    {\"defaultWrappingKey\":\"master2\",\"retrying\": {\"attemptTimeoutMills\":20000, \"maxAttempts\":3}, \"wrappingKeys\": \
+    [{\"alias\":\"master1\", \"passphrase\": {\"configSecret\": {\"encryptedSecret\" :\"ZAmGcppwqUNsViSE06b801DVARbD9wT7SZnYP03WKZFOnC9KTsII04pWWq5Je2CUZdP+xIsHFUO3RkZVwhwnyQhWoxcW6JEo\"}},\"salt\": \
+    {\"configSecret\":{\"encryptedSecret\":\"cvNVJ4D7fNGm59Uh3Tlhi0W3pToDxl7vykUdZaxEuEPGUMdmu7G+xTEfl0CcC2R9aWl6FgrzmX5x+qCGNElUi/PBA40EGjEU\"}}}, \
+    {\"alias\":\"master2\",\"passphrase\":{\"configSecret\": {\"encryptedSecret\":\"gzXRjE4AoZSW8GQhcqbw0shQfwWGy5O8alNDqXaqTUzyUnLBVUXLwhv+uAEZzecQixiaEaNtfOvuftYW9NjwwRyf6m4KZIag\"}},\"salt\": \
+    {\"configSecret\":{\"encryptedSecret\":\"flvAnf5G/aRmDIaahXHjftEpc0odufhzZsuiipxtQTFV+pccB+RHnIqle26jjDlDUBfn3HPbVezKhpRHL/NP5M+cX/rU15DE\"}}}]}, \
+    \"retrying\":{\"maxAttempts\": {\"default\": 3},\"waitBetweenMills\":{\"default\":[200]}}}}","schemaVersion": {"major":1, "minor":0}}' \
+    "$REST_API_URL/config"
+   ```
+   {{% /tab %}}
+   {{% tab name="PowerShell" %}}
+
+   ```shell
+  Invoke-RestMethod -SkipCertificateCheck -Headers @{Authorization=("Basic {0}" -f ${REST_API_USER}:${REST_API_PASSWORD})} -Method Put -Uri "$REST_API_URL/config" -Body (ConvertTo-Json -Depth 4 @{
+   {
+     "config": "{
+        "caching":{
+          "expireAfterAccessMins":{
+            "default":60
+          },
+          "maximumSize":{
+            "default":10000
+          }
+        },
+        "hsm":{
+          "defaultWrappingKey": "master1",
+          "retrying":{
+            "attemptTimeoutMills": 20000,
+            "maxAttempts": 3
+          },
+          "wrappingKeys":[{
+            "alias": "master1",
+            "passphrase":{
+              "configSecret":
+              {
+                "encryptedSecret": "gzXRjE4AoZSW8GQhcqbw0shQfwWGy5O8alNDqXaqTUzyUnLBVUXLwhv+uAEZzecQixiaEaNtfOvuftYW9NjwwRyf6m4KZIag"
+              }
+            },
+            "salt":{
+              "configSecret":
+              {
+                "encryptedSecret": "cvNVJ4D7fNGm59Uh3Tlhi0W3pToDxl7vykUdZaxEuEPGUMdmu7G+xTEfl0CcC2R9aWl6FgrzmX5x+qCGNElUi/PBA40EGjEU"
+              }
+            },
+          },
+          {
+            "alias": "master2",
+            "passphrase":{
+              "configSecret":
+              {
+                "encryptedSecret": "ZAmGcppwqUNsViSE06b801DVARbD9wT7SZnYP03WKZFOnC9KTsII04pWWq5Je2CUZdP+xIsHFUO3RkZVwhwnyQhWoxcW6JEo"
```

Comment:
I think we should describe how a user would generate this. Also perhaps we need to point out this is the config for a Corda internal master key, and also provide an example for an externally managed one.

Reply:
@simon-johnson-r3 Could you provide me with sample values? `vaultPath` and `vaultKey` I think?

Reply:
Example added in https://github.com/corda/corda-docs-portal/pull/2047/commits/42466aad242375a9636bdeeac210ada7f6cbce91